packages/server/src/api/routes/tests/search.spec.ts

```diff
@@ -1797,4 +1837,40 @@ describe.each([
           { two: "foo", other: [{ _id: otherRows[0]._id }] },
         ]))
     })
+
+  // lucene can't count, and in memory there is no point
+  !isLucene &&
+    !isInMemory &&
+    describe("row counting", () => {
+      beforeAll(async () => {
+        table = await createTable({
+          name: {
+            name: "name",
+            type: FieldType.STRING,
+          },
+        })
+        await createRows([{ name: "a" }, { name: "b" }])
+      })
+
+      it("should be able to count rows when option set", () =>
+        expectSearch({
+          countRows: true,
+          query: {
+            notEmpty: {
+              name: true,
+            },
+          },
+        }).toHaveProperty([{ key: "totalRows", value: 2 }, { key: "rows" }]))
```

Comment:
Is this not just the jest `toMatch` assertion? Could we implement `expectSearch({ ... }).toMatch({ totalRows: 2, rows: expect.any(Array) })` so it fits more naturally into jest?